src/github/client.py

```diff
@@ -42,7 +42,7 @@ def merge_pull_request(owner: str, repository: str, number: int, title: str, bod
     # we add the PR number to match Github's default squash and merge title style
     # which we rely on for code review tests.
     title_with_number = f"{title} (#{number})"
-    pr.merge(commit_title=title_with_number, commit_message=body, merge_method="squash")  # type: ignore
```

Comment:
as per https://docs.github.com/en/graphql/reference/input-objects#enablepullrequestautomergeinput
<img width="727" alt="image" src="https://github.com/Asana/SGTM/assets/34071587/e3b02c03-7299-4425-8985-33e23e467ffc">

merge_method is ignored when using a merge queue

Reply:
Does this still enable automerge to work in directories that don't have merge queues turned on? The docs aren't super clear 🤔

Reply:
hmm looks like it will have the same functionality as clicking the "Click Enable auto-merge" button on the github UI https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/incorporating-changes-from-a-pull-request/automatically-merging-a-pull-request

I don't think we have this button turned on for codez but should work for repositories that do have that option

Reply:
We should verify that this works with repositories that have direct merge configuration as well.

Reply:
currently testing in the testing-sgtm repo

Reply:
looks like this won't work for repos without "automerge" turned on, updated to manual merge if trying to automerge fails (test run https://github.com/Asana/testing-sgtm/pull/4)